tests/simple.rs

```diff
@@ -4,16 +4,12 @@ mod common;
 
 #[tokio::test]
 async fn test_integration_simple() -> eyre::Result<()> {
```

Comment:
what's the difference between this test and `test_integration_simple_isthmus_transition`? feel the tests are the same

Reply:
This one starts with a chain that is v3 and test_integration_simple_isthmus_transition transitions from a chain that uses v3 to one that uses v4 at block 5.